### ISSUE TITLE
Back button doesn't work in search

### ISSUE THREAD

**Issue by pdpinch** [mitodl/ocw-www#94]
### Steps to Reproduce

1. Go to https://ocwnext.odl.mit.edu/search/
2. Search for "Monday" (for example)
3. Browser URL updates to https://ocwnext.odl.mit.edu/search/?q=monday
2. Search for "Tuesday" (for example)
3. Browser URL updates to https://ocwnext.odl.mit.edu/search/?q=tuesday
3. Click the back button

### Expected Behavior

- You should return to the first search, for "Monday" and the URL https://ocwnext.odl.mit.edu/search/?q=monday

### Actual Behavior

- browser returns to whatever page it was on before navigating to https://ocwnext.odl.mit.edu/search/

**Comment by noisecapella** [comment]
@pdpinch Do we want this behavior for open-discussions search too?

**Comment by pdpinch** [comment]
Yes, especially if it’s easier to keep it consistent. 

The back button should always work, right?

**Comment by noisecapella** [comment]
Yes, since the change is in course-search-utils it would be easier to do both. Though it won't update for open-discussions until we choose to upgrade versions

**Comment by noisecapella** [comment]
The basic functionality is working but it looks like clicking the back button to leave the search page is broken, so I'm looking into fixing that
